examples/python/testcase1.py

```diff
@@ -9,22 +9,14 @@
 
 # GENERAL PACKAGE IMPORT
 # ----------------------
-import requests
-import numpy as np
-from examples.python.custom_kpi import custom_kpi_calculator as kpicalculation
-import json,collections
-import pandas as pd
+import sys
```

Comment:
The doc string description of this module ``testcase1.py`` is the same as ``interface.py`` and needs to be corrected.

Reply:
I updated the doc string

Reply:
Great thanks!